### DISCUSSION TITLE
Grove - OLED Display 0.96" (SSD1315) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1181] [Q&A]
# Grove - OLED Display 0.96" (SSD1315) | Seeed Studio Wiki

Grove - OLED Display 0.96" (SSD1315)

https://wiki.seeedstudio.com/Grove-OLED-Display-0.96-SSD1315/

**Comment by Morgaul** [comment]
I was able to run the display with the following settings:
Visual Studio Code + PlatformIO + olikraus/U8g2@^2.35.19 + ESP32-S3

#define SSD1315_ADDR 0x78
#define I2C_SPEED 400E3
#define mcuSDA 8
#define mcuSCL 9

U8G2_SSD1306_128X64_NONAME_F_HW_I2C u8g2(U8G2_R0);

Wire.begin(mcuSDA, mcuSCL, I2C_SPEED);
u8g2.setI2CAddress(SSD1315_ADDR);
u8g2.begin();

I hope I was able to help someone.
